src/jailer/src/env.rs

```diff
@@ -609,6 +609,80 @@ mod tests {
                 metadata.permissions().mode(),
                 libc::S_IFCHR | libc::S_IRUSR | libc::S_IWUSR
             );
+
+            // Trying to create again the same device node is not allowed.
+            assert_eq!(
+                format!("{}", env.mknod_and_own_dev(dev, major, minor).unwrap_err()),
+                format!(
+                    "Failed to create {}\u{0} via mknod inside the jail: File exists (os error 17)",
+                    dev_str
+                )
+            );
+            // Remove the device node.
+            fs::remove_file(dev_str).expect("Could not remove file.");
         }
     }
+
+    #[test]
+    fn test_prepare_for_chroot() {
+        // Create a standard environment.
+        let arg_parser = build_arg_parser();
+        let mut args = arg_parser.arguments().clone();
+        // Not using here `ArgVals::new()` because `fs::copy()` from `prepare_for_chroot()`
+        // sees `/proc/cpuinfo` as an empty file and doesn't copy anything to destination.
+        let some_arg_vals = ArgVals {
+            node: "1",
+            id: "bd65600d-8669-4903-8a14-af88203add38",
+            exec_file: "file1",
```

Comment:
Can`t we use `TempFile` for zzns and file1  and TempDir for `chroot_base`?

Reply:
:+1: done.